copulas/multivariate/vine.py

```diff
@@ -272,16 +272,13 @@ def _sample_row(self):
                         copula_type = current_tree[current_ind].name
                         copula = Bivariate(copula_type=CopulaTypes(copula_type))
                         copula.theta = current_tree[current_ind].theta
-                        derivative = copula.partial_derivative_scalar
```

Comment:
This is re-implementing the conditional PPF using a numerical approximation instead of using the (potentially) closed form solution provided by the Copulas. Why is it doing this? I have no idea, but it's slower and less accurate.